src/view/com/util/UserAvatar.tsx

```diff
@@ -313,7 +313,8 @@ let EditableUserAvatar = ({
 
       onSelectNewAvatar(croppedImage)
     } catch (e: any) {
-      if (!String(e).includes('Canceled')) {
+      // Don't log errors for cancelling selection to sentry on ios or android
+      if (!String(e).toLowerCase().includes('cancel')) {
```

Comment:
The error here is different depending on platform. Funny enough, on one platform it's spelled `canceled` and on the other platform it's spelled `cancelled`. Looking to see if it includes `cancel` will detect the error in both cases.